### PR TITLE
Realm settings(login tab): Add login tests

### PR DESCRIPTION
**Pull request by jenny-s51**
Adds login tests: 

https://docs.google.com/spreadsheets/d/1eCngdn3wmMeQ0u5u-eKYJo1XdT2fHvi7ki51mrcu-Ys/edit#gid=1613790516&range=443:446

### CODE REVIEW

**Review by mfrances17 — CHANGES_REQUESTED** [review]
I think you missed the test for the duplicate emails enabled/disabled switch (445)

**Comment by jenny-s51** [comment]
Marking as draft -- these tests revealed some bugs in how these switches are currently implemented. Investigating a fix! Plan to have a PR up shortly with an updated implementation for saving switch values.

File: src/realm-settings/LoginTab.tsx [hunk 1]
```diff
@@ -58,12 +78,16 @@ export const RealmSettingsLoginTab = ({
                 <Switch
                   id="kc-user-reg-switch"
                   data-testid="user-reg-switch"
-                  name="registrationAllowed"
+                  value={
+                    value
+                      ? t("common:on").toLowerCase()
```
Comment (jonkoops) [inline src/realm-settings/LoginTab.tsx:83]: Why do we need to use a translated label as the value of this? Should this not be the same value regardless of language?

File: src/realm-settings/PoliciesTab.tsx [hunk 2]
```diff
@@ -65,6 +65,7 @@ export const PoliciesTab = () => {
 
   const saveStatus = async () => {
     const switchValues = form.getValues();
+    console.log(switchValues);
```
Comment (jonkoops) [inline src/realm-settings/PoliciesTab.tsx:68]: Stray console log.
```suggestion
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/realm-settings/LoginTab.tsx [hunk 3]
```diff
@@ -89,11 +109,16 @@ export const RealmSettingsLoginTab = ({
                   id="kc-forgot-pw-switch"
                   data-testid="forgot-pw-switch"
                   name="resetPasswordAllowed"
+                  value={
+                    value
+                      ? t("common:on").toLowerCase()
+                      : t("common:off").toLowerCase()
```
Comment (jonkoops) [inline src/realm-settings/LoginTab.tsx:115]: The value should not be translated here right? I suppose these should be replaced with static values.

File: src/realm-settings/LoginTab.tsx [hunk 4]
```diff
@@ -118,12 +143,16 @@ export const RealmSettingsLoginTab = ({
                 <Switch
                   id="kc-remember-me-switch"
                   data-testid="remember-me-switch"
-                  name="rememberMe"
+                  value={
+                    value
+                      ? t("common:on").toLowerCase()
+                      : t("common:off").toLowerCase()
```
Comment (jonkoops) [inline src/realm-settings/LoginTab.tsx:149]: Some here, the value is again translated.

File: src/realm-settings/LoginTab.tsx [hunk 5]
```diff
@@ -152,16 +181,16 @@ export const RealmSettingsLoginTab = ({
                 <Switch
                   id="kc-email-as-username-switch"
                   data-testid="email-as-username-switch"
-                  name="registrationEmailAsUsername"
+                  value={
+                    value
+                      ? t("common:on").toLowerCase()
+                      : t("common:off").toLowerCase()
```
Comment (jonkoops) [inline src/realm-settings/LoginTab.tsx:187]: Same here.

File: src/realm-settings/LoginTab.tsx [hunk 6]
```diff
@@ -186,12 +215,16 @@ export const RealmSettingsLoginTab = ({
                 <Switch
                   id="kc-login-with-email-switch"
                   data-testid="login-with-email-switch"
-                  name="loginWithEmailAllowed"
+                  value={
+                    value
+                      ? t("common:on").toLowerCase()
+                      : t("common:off").toLowerCase()
```
Comment (jonkoops) [inline src/realm-settings/LoginTab.tsx:221]: And here.

File: src/realm-settings/LoginTab.tsx [hunk 7]
```diff
@@ -259,11 +287,16 @@ export const RealmSettingsLoginTab = ({
                   id="kc-verify-email-switch"
                   data-testid="verify-email-switch"
                   name="verifyEmail"
+                  value={
+                    value
+                      ? t("common:on").toLowerCase()
+                      : t("common:off").toLowerCase()
```
Comment (jonkoops) [inline src/realm-settings/LoginTab.tsx:293]: And here again.

File: src/realm-settings/LoginTab.tsx [hunk 8]
```diff
@@ -295,12 +328,16 @@ export const RealmSettingsLoginTab = ({
                 <Switch
                   id="kc-edit-username-switch"
                   data-testid="edit-username-switch"
-                  name="editUsernameAllowed"
+                  value={
+                    value
+                      ? t("common:on").toLowerCase()
+                      : t("common:off").toLowerCase()
```
Comment (jonkoops) [inline src/realm-settings/LoginTab.tsx:334]: Also here.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by jonkoops** [comment]
@jenny-s51 can you also rebase this PR on `main` so the tests can pass again?

**Review by jonkoops — APPROVED** [review]
Looking good, let's see if the tests will pass.

**Pull request merged**